roachpb/api.go

```diff
@@ -770,7 +770,10 @@ func (*AdminTransferLeaseRequest) flags() int { return isAdmin | isAlone }
 func (*HeartbeatTxnRequest) flags() int       { return isWrite | isTxn }
 func (*GCRequest) flags() int                 { return isWrite | isRange }
 func (*PushTxnRequest) flags() int            { return isWrite }
-func (*RangeLookupRequest) flags() int        { return isRead | isTxn }
+
+// TODO(tschottdorf): why does RangeLookupRequest have the isTxn flag?
+func (*RangeLookupRequest) flags() int { return isRead | isTxn }
```

Comment:
The `isTxn` flag makes no sense here and should be removed. I tracked down the one usage in `DistSender` and it completely orthogonal to `RangeLookupRequest`. Remove the flag.